packages/mjml-column/README.md

```diff
@@ -1,9 +1,20 @@
 ## mj-column
 
-Columns enable you to horizontally organize the content within your sections. They must be located under `mj-section` tags in order to be considered by the engine.
-To be responsive, columns are expressed in terms of percentage.
+Columns are key to MJML's easy responsiveness.
 
-Every single column has to contain something because they are responsive containers, and will be vertically stacked on a mobile view. Any standard component, or component that you have defined and registered, can be placed within a column – except `mj-column` or `mj-section` elements.
+Columns contain vertically-structured content.
+They must be inside `mj-section` elements to be considered by MJML.
+An `mj-column` with widths expressed in percentages is responsive.
+An `mj-column` with fixed width (pixels) is not responsive.
```

Comment:
What this is totally wrong ?

Reply:
Glad you caught it. Which?

Reply:
Column in px / % are just behaving the same it's just a convenience